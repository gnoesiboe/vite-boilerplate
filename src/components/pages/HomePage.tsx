import React, { VFC } from 'react';
import { Helmet } from 'react-helmet';
import { composeMetaTitle } from '../../utilities/pageMetaUtilities';

const HomePage: VFC = () => (
    <>
        <Helmet>
            <title>{composeMetaTitle('Home')}</title>
        </Helmet>
        @todo implement homepage
    </>
);

export default HomePage;
