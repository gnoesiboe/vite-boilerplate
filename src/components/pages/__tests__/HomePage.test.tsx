import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../HomePage';

describe('Homepage', () => {
    it('should match the snapshot', () => {
        const component = renderer.create(<HomePage />);

        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});
