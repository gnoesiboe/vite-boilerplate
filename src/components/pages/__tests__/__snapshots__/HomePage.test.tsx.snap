// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should match the snapshot 1`] = `"@todo implement homepage"`;
